sdk/servicebus/Azure.Messaging.ServiceBus/tests/Infrastructure/ServiceBusLiveTestBase.cs

```diff
@@ -40,7 +40,6 @@ protected ServiceBusClient CreateClient(int tryTimeout = DefaultTryTimeout)
                     RetryOptions = new ServiceBusRetryOptions
                     {
                         TryTimeout = TimeSpan.FromSeconds(tryTimeout),
-                        MaxRetries = 0
```

Comment:
This was a copy paste error added in https://github.com/Azure/azure-sdk-for-net/pull/20940